exercises/change/change.spec.js

```diff
@@ -0,0 +1,80 @@
+import Change from './change';
+
+describe('Change', () => {
+  test('change for 1 cent', () => {
+    const change = new Change();
+    const result = change.calculate([1, 5, 10, 25], 1);
+    expect(result).toEqual([1]);
+  });
+
+  xtest('return a single coin', () => {
+    const change = new Change();
+    const result = change.calculate([1, 5, 10, 25, 100], 25);
+    expect(result).toEqual([25]);
+  });
+
+  xtest('multiple coins coin', () => {
```

Comment:
Should this read `multiple coin change`, as in the [canonical file](https://github.com/exercism/problem-specifications/blob/master/exercises/change/canonical-data.json#L18)?